crates/webidl/src/lib.rs

```diff
@@ -211,12 +211,22 @@ impl<'a> WebidlParse<&'a webidl::ast::NonPartialInterface> for webidl::ast::Exte
                 webidl::ast::ArgumentListExtendedAttribute { arguments, name },
             ) if name == "Constructor" =>
             {
-                add_constructor(&*arguments);
+                add_constructor(arguments, &interface.name);
             }
             webidl::ast::ExtendedAttribute::NoArguments(webidl::ast::Other::Identifier(name))
                 if name == "Constructor" =>
             {
-                add_constructor(&[] as &[_]);
+                add_constructor(&[] as &[_], &interface.name);
```

Comment:
Is that `&[] as &[_]` cast necessary?

Reply:
No it is not, I cleaned it up in my follow-up work on static methods and attributes, which is just pending on this being merged (leverages the refactoring). I can backport the cleanup if you would like.

Reply:
Nah it's fine, I didn't realize it wasn't even introduced in this PR. I'll merge this and then we can fix it in a follow up. Thanks!